cocos2d/core/layers/CCLayerCanvasRenderCmd.js

```diff
@@ -244,7 +245,7 @@
             + (0 | curColor.b) + ", 1)");  //TODO: need cache the color string
 
         wrapper.setTransform(this._worldTransform, scaleX, scaleY);
-        context.fillRect(0, 0, locWidth * scaleX, -locHeight * scaleY);
+        context.fillRect(0, 0, locWidth , -locHeight );
```

Comment:
I don't understand how can we preserve the same rendering result after removing scaleX & scaleY